### ISSUE TITLE
support multiple languages - internalization (i18n) 

### ISSUE THREAD

**Issue by ozayduman** [Pitayan/gatsby-theme-pitayan#22]
It would be great to support multiple languages

**Comment by daiyanze** [comment]
Agree. Multi-lang support seems quite a good feature. Do you already have the idea of how to apply multiple

**Comment by ozayduman** [comment]
There is a popular English learning site called [engoo.com](https://engoo.com/app/materials/business/RWVinsOrEeiE8w-LaNCm_w#course-KIgMYsYNEeikgiuzpZcIlA) Likewise, in your theme at the top right corner language selection buton would be shown if the site supports multi-lang. The default language can be determined as user's browser language. Also fields like time to read should be localized. I think the structure of the content folder might look like this:
```
your-site
  ├── content/
  ├── en/
  │ ├── authors/
  │ ├── posts/
  │ └── site/
  ├── tr/
  │ ├── authors/
  │ ├── posts/
  │ └── site/
  ├── de/
  │ ├── authors/
  │ ├── posts/
  │ └── site/
..
  ├── src/
  │ └── assets/
```
BTW, I love this theme and I appreciate your great work

**Comment by daiyanze** [comment]
Thanks for the great idea. 

I think there'll be at least the following things to be optimized for multi-lang:
- Hard-coded template components (provide configurations for localization)
- A button (popover) to allow switch between languages

We also need a good plugin to bring multi-lang into this theme.
https://www.gatsbyjs.com/docs/how-to/adding-common-features/localization-i18n/

Currently, I'm busy with some other projects. Don't really have the bandwidth with this feature at the moment. I'll put this feature to the roadmap after "making Tailwindcss theme preset". (PR welcome :D)
